### DISCUSSION TITLE
Transfer Incomplete

### DISCUSSION THREAD

**Discussion by SleeplessinDC** [iterate-ch/cyberduck#15843] [Q&A]
When downloading files from the server to my desktop, most of my sites completed without a problem but two didn't. They went down to the zero second remaining, then ended with "Transfer Incomplete". I tried each of them again twice and the same thing happened. I cleared the transfer log, quit Cyberduck, and tried one of them that did complete. I cleared the transfer log again and quit Cyberduck. Then tried to download the other site. But it still went to zero then quit with "Transfer Incomplete". How can I find out why it didn't complete? Is there a way to keep a record of the activity to find out what is missing?

**Comment by tatze96** [comment]
Unfortunately, there isn't an easy way to find out what wasn't transferred apart from [resuming](https://docs.cyberduck.io/cyberduck/transfer/#resume) the transfer and subsequently reinitiating the transfer. On resume Cyberduck will skip the items that match in size and checksum.

- Which protocol is in use?
- Do you get an error message alongside the *Transfer Incomplete* indication? If so, attach a screenshot.

Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the *Transfer Incomplete* message, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15843 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file and screenshot here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
